ynput/ayon-unreal
# Check for existing UAssets when loading data

## Changelog Description
Resolve https://github.com/ynput/ayon-unreal/issues/42

## Additional info
we may consider adding the version as asset name.

https://github.com/user-attachments/assets/386615ee-0002-4041-bba9-e590f07f678d


## Testing notes:
1. Launch Unreal
2. Test all the loaders
3. Should be no error message
4. Updating assets or Set Version to the asset in scene inventory
5. Should be still creating the exclusive version and loaded with the relevant version of asset
6. Test with load function in different version
7. Still loading the asset with exclusive version as expected
8. Move the asset out of the version folder
9. Load it again
10. The asset would be moved back to the version folder and updated to the related version.
